separated-interface/README.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: pattern
+title: Separated Interface
+folder: separated-interface
+permalink: /patterns/separated-interface/
+categories: Architectural
```

Comment:
In a sense it's architectural, but I would put it under `Structural` since this the scope is so small.